src/pivot-table/components/Wrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useStyleContext } from "../contexts/StyleProvider";
+import { Disclaimer } from "./Disclaimer";
+import { PivotTableProps, StickyPivotTable } from "./PivotTable";
+
+export const Wrapper = (props: PivotTableProps): JSX.Element => {
+  const {
+    layoutService: { hasLimitedData },
+  } = props;
+  const styleService = useStyleContext();
+
+  return (
+    <div>
```

Comment:
I think fragment should be a good choice here

Reply:
Fixed